base/inc/widgets/base-slider.class.php

```diff
@@ -137,7 +137,7 @@ function video_form_fields(){
 	function slider_settings( $controls ){
 		return array(
 			'pagination' => true,
-			'speed' => $controls['speed'],
+			'speed' => $controls['speed'] !== 0 ? $controls['speed'] : 1,
```

Comment:
It would be neater using an `empty` here. That would cover `0` and a few other possible edge cases. Not essential, but worth changing unless it'll cause any problems you can think of.